### ISSUE TITLE
Outdated OSAL for CFE 6.5.0a

### ISSUE THREAD

**Issue by vwchu** [yusend/coreflightexec#1]
OSAL version outdated `4.1.1`. README states:

```
Software Required:

 Operating System Abstraction Layer 4.2.0 or higher
    Can be obtained at http://sourceforge.net/projects/osal
     or github.com/nasa/osal
```

**coreflightexec version:** 6.5.0a
**OSAL**: 4.1.1
**Operating System:** Linux Ubuntu 12.04 32-bit

Results in the following error when compiling:

```
make -C link
make[2]: Entering directory `/home/vagrant/coreflightexec/build/cpu1/cfe/link'
gcc  -o core-linux.bin ../psp/psp.o ../osal/osal.o ../es/es.o ../sb/sb.o ../time/time.o ../evs/evs.o ../fs/fs.o  ../tbl/tbl.o -m32 -Wl,-export-dynamic -lm -lpthread -ldl -lrt
../psp/psp.o: In function `main':
/home/vagrant/coreflightexec/psp/fsw/pc-linux/src/cfe_psp_start.c:358: undefined reference to `OS_IdleLoop'
/home/vagrant/coreflightexec/psp/fsw/pc-linux/src/cfe_psp_start.c:372: undefined reference to `OS_DeleteAllObjects'
../psp/psp.o: In function `CFE_PSP_SigintHandler':
/home/vagrant/coreflightexec/psp/fsw/pc-linux/src/cfe_psp_start.c:393: undefined reference to `OS_ApplicationShutdown'
../time/time.o: In function `CFE_TIME_TaskInit':
/home/vagrant/coreflightexec/cfe/fsw/cfe-core/src/time/cfe_time_task.c:442: undefined reference to `OS_TimeBaseGetIdByName'
/home/vagrant/coreflightexec/cfe/fsw/cfe-core/src/time/cfe_time_task.c:446: undefined reference to `OS_TimerAdd'
collect2: ld returned 1 exit status
make[2]: *** [core-linux.bin] Error 1
make[2]: Leaving directory `/home/vagrant/coreflightexec/build/cpu1/cfe/link'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/home/vagrant/coreflightexec/build/cpu1/cfe'
make: *** [cfs_build_core] Error 2
```

**Comment by vwchu** [comment]
Possibly cause, OSAL version outdated `4.1.1`. README states:

```
Software Required:

 Operating System Abstraction Layer 4.2.0 or higher
    Can be obtained at http://sourceforge.net/projects/osal
     or github.com/nasa/osal
```

**Comment by vwchu** [comment]
Resolved with bfa6f6f43ccb67d8af72d4ae2e009d9ef6ceb230

**Comment by vwchu** [comment]
Resolved with 07f16ae86500de8ee9f75de66bfa34d90440d5ee
